core/src/main/java/io/aiven/klaw/controller/EnvsClustersTenantsController.java

```diff
@@ -336,23 +334,6 @@ public ResponseEntity<KwTenantModel> getMyTenantInfo() {
         envsClustersTenantsControllerService.getMyTenantInfo(), HttpStatus.OK);
   }
 
-  @RequestMapping(
-      value = "/getAclCommands",
-      method = RequestMethod.GET,
-      produces = {MediaType.APPLICATION_JSON_VALUE})
-  public ResponseEntity<AclCommands> getAclCommand() {
-    return new ResponseEntity<>(
-        envsClustersTenantsControllerService.getAclCommands(), HttpStatus.OK);
```

Comment:
This getAclCommands is referenced in helpwizard.html and addCluster.html , I can see they are also removed from the HTML files, but they seem unrelated to saas?

Reply:
yea those acl commands were mostly for saas related. Removed them now as Infra admin will create those required acls anyways, and not really required to mention in the addclusters page.